chapters/en/slides/module1/module1_03.Rmd

```diff
@@ -32,15 +31,15 @@ When learning about data visualization, it is helpful to distinguish between the
 
 Notes: 
 
-There is a plethora of visualization packages in Python.
```

Comment:
This always confuses me. Isn't it a single "plethora"? Just as I would say "There is a bucket of toys over there" rather than "There are a bucket of toys over there"? English is weird https://englishlessonsbrighton.co.uk/there-is-a-lot-of-vs-there-are-a-lot-of/